src/error/__tests__/formatError-test.ts

```diff
@@ -6,7 +6,6 @@ import { GraphQLError } from '../GraphQLError';
 
 describe('formatError: default error formatter', () => {
   it('uses default message', () => {
-    // $FlowExpectedError[incompatible-call]
```

Comment:
@saihaj Please replace instances of `$FlowExpectedError` with `@ts-expect-error`.